src/bin/upgrade/main.rs

```diff
@@ -53,7 +53,11 @@ Dev, build, and all target dependencies will also be upgraded. Only dependencies
 supported. Git/path dependencies will be ignored.
 
 All packages in the workspace will be upgraded if the `--all` flag is supplied. The `--all` flag may
-be supplied in the presence of a virtual manifest."
+be supplied in the presence of a virtual manifest.
+
+If the '--to-lockfile' flag is supplied, all dependencies will be upraged to the currently locked
+version as recorded in the local lock file. The local lock file must be present and up to date if
+this flag is passed."
```

Comment:
could you clarify in the docs what you mean by `MUST`, e.g. will it result in an error, how does it work with the `offline` mode, etc